README.adoc

```diff
@@ -184,33 +206,39 @@ include::finish/src/main/java/io/openliberty/guides/jpaguide/dao/EventDao.java[t
-context will be notified that the transaction is committing and the persistence provider, EclipseLink,
-will flush the changes to the database.
+`@PersistenceContext` annotation. Since the stateless session bean is configured to manage the global
+transactions, the EJB container will ensure that the required methods are executed within the boundaries
+of an active global transaction, which is why it is not necessary to explicitly begin, commit or rollback
+transactions. At the end of a session bean method invocation, the persistence context will be notified
+that the transaction is committing, and will flush any changes made to entity instances it is managing
+to the database.
 
 The `EventDao.java` class has a method for each CRUD operation, so let's break them down:
 
 * The `createEvent()` method persists an instance of the `Event` entity class to the datastore by
-calling the `persist()` method on an `EntityManager` instance. The argument passed into the `persist()`
-method must be an instance of an entity class, otherwise an `IllegalArgumentException` will be thrown.  
+calling the `persist()` method on an EntityManager instance. The event objects state will be 
+changed from new to managed and changes to it are now tracked by the EntityManager. 
 
 * The `readEvent()` method will return an instance of the `Event` entity class with the specified primary 
 key by calling the `find()` method on an `EntityManager` instance. The `find()` method accepts both the entity 
-class and the primary key as arguments. If the event instance is not found, `Null` will be returned. Similarily,
-the `readAllEvents()` method returns a list of instances of the `Event` entity class using a query
-specified in the `@NamedQuery` annotation on the `Event` class. 
+class and the primary key as arguments. If the state of the entity being searched for is managed, the event 
+object will be returned as is and no retrieval from the database is required. But, if it is not already managed,
+a new entity object will be constructed with data retrieved from the database and returned in a managed state.
+If the event instance is not found, `Null` will be returned.
+
+* The `readAllEvents()` method demonstrates an alternate way to retrieve event objects from the database.
+This method returns a list of instances of the `Event` entity class using a query specified in the `@NamedQuery` 
+annotation on the `Event` class. 
 
-* The `updateEvent()` method updates an instance of the `Event` entity class in the database. The 
-`EntityManager` automatically tracks entity objects in the persistence context for changes and synchronizes
-them with the database. The `merge()` method will copy the entity supplied as an argument into a new 
-`Event` instance that will be added to the persistence context so that its changes can be tracked.
+* The `updateEvent() method creates a managed instance of a detached entity object. The EntityManager 
```

Comment:
Might want to explain why the entity is detached, and explain that the merge operation not only returns a managed instance of the merge() argument but also includes changes made to the detached entity's loaded fields.